### DISCUSSION TITLE
Fermata auto does not appear on Android Auto

### DISCUSSION THREAD

**Discussion by rubensasouto** [AndreyPavlenko/Fermata#357] [General]
My phone updated the version and fermata auto disappeared from Android Auto. I updated today to version 1.9.7 and still only fermata control appears. Does anyone know the solution?

**Comment by maripic69** [comment]
Чакаме решение!

**Comment by prw19** [comment]
Still same here. 
Samsung S23+ Android 14
Android auto version 11.1.xx 

Please help us!

**Comment by abdo1991yogi** [comment]
I updated fermata auto to version 1.9.8 and it still does not appear in Android auto

**Reply by AndreyPavlenko** [reply]
Please, do not expect a fix on the app side. Before Android 14 a special installer used to enable the app. Since A14 it does not work any more.

**Reply by jrdennyjr** [reply]
So much for purchasing the pro version. Screw us...

**Reply by prw19** [reply]
Oke I want a pro version. Is the problem with Pro version fixed?

**Reply by AndreyPavlenko** [reply]
There is no any Pro version.

**Reply by maripic69** [reply]
Ще има ли някакво решение за Fermata auto в Android 14!?

**Reply by AndreyPavlenko** [reply]
https://github.com/AndreyPavlenko/Fermata/issues/327#issuecomment-1915058419
https://github.com/AndreyPavlenko/Fermata/issues/327#issuecomment-1916776751

**Comment by globalconnectone** [comment]
same here updated today to UI 6.0 Android 14 on my Samsung Galaxy Fold 4 and fermata auto disappeared from Android Auto.
Andrey how we make it to work, please advise or provide solution, i know you can make it happened . Thank you in advance.

**Comment by Unified-Supreme** [comment]
Hi Guys Same over here, i own an S24 Ultra and i am unable to make it run, hope it gets fixed soon, i have 6.1 One UI version and android 14 with Android Auto 11.1
